Sources/UINotificationView.swift

```diff
@@ -124,9 +145,6 @@ open class UINotificationView: UIView {
         imageView.image = notification.content.image
         imageView.isHidden = notification.content.image == nil
         
-        let chevronImageWidth = chevronImageView.image?.size.width ?? 0
-        containerStackView.spacing = imageView.isHidden ? -chevronImageWidth : containerStackViewDefaultSpacing
```

Comment:
@AvdLee, you added those lines when working on the action.
I can't figure out why we need them. Could you tell me more.
FYI, when the `imageView` is hidden, my button overlaps the chevron if I leave them.

Reply:
I guess this code is needed for when you show a notification with an image in it ánd the notification has a tap action attached so we show the chevron. 

Did you verify that the chevron is still displayed correctly with an image in the notification and a tap action attached?

Reply:
Yeah, it works just fine for some reason.
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-03-11 at 09 55 01](https://user-images.githubusercontent.com/4613643/76399181-7ea51a80-637e-11ea-9646-136a9ddce448.png)

Reply:
@Boris-Em I guess we're good then! Might as well verify in Coyote as an implementation example. However, we would run into this either way. I'm fine keeping it like this and see whether we run into issues! 🙂